napalm/ios/ios.py

```diff
@@ -506,15 +507,37 @@ def commit_config(self, message="", revert_in=None):
 
         If merge operation, perform copy <file> running-config.
         """
-        if revert_in is not None:
-            raise NotImplementedError(
-                "Commit confirm has not been implemented on this platform."
-            )
+        CISCO_TIMER_MIN = 1
+        CISCO_TIMER_MAX = 120
+        ARCHIVE_DISABLED_MESSAGE = (
+            "For Cisco devices, revert_in requires 'archive' feature to be enabled."
+        )
+
         if message:
             raise NotImplementedError(
                 "Commit message not implemented for this platform"
             )
 
+        if revert_in is not None:
+            if not self._check_archive_feature():
+                raise CommitConfirmException(ARCHIVE_DISABLED_MESSAGE)
+            elif not CISCO_TIMER_MIN * 60 <= revert_in <= CISCO_TIMER_MAX * 60:
+                msg = (
+                    "For Cisco devices revert_in between {} and {} seconds, this will round "
+                    "down to the nearest minute".format(
+                        CISCO_TIMER_MIN * 60, CISCO_TIMER_MAX * 60
+                    )
```

Comment:
Could it be a little clearer to say "For Cisco IOS devices revert_in is rounded down to the nearest minute, pass revert_in as a multiple of 60"?

Reply:
Line 648